### DISCUSSION TITLE
Agent Workspace Transition

### DISCUSSION THREAD

**Discussion by ftzortsos** [cloudbees/zendesk-java-client#472] [General]
Hello, 

My tech team, used this Library to create a custom api to export data from our Zendesk.

I have one question, no tech side, We are going to activate the Agent Workspace, in our Zendesk platform, and we want to make sure that this build will still be functional, and we will keep retrieving the data.

I also asked Zendesk Support to provide feedback, but they referred to ask here.

Does anyone work this client with Agent Workspace?

Many Thanks

**Comment by aheritier** [comment]
We don't use them (yet) on our side (cloudbees). 

I don't think it should impact the APIs because it's more a UI feature

**Comment by aheritier** [comment]
FYI in our staging environment where we run our integration tests for theses APIs we enabled the Agent Workspace feature and we didn't notice any issue
